Source/Protocols/Request.swift

```diff
@@ -2,28 +2,86 @@ import Foundation
 import Argo
 import Result
 
+/**
+  Used to represent a request that doesn't return any JSON.
```

Comment:
I would probably try to stay away from past tense ("used" even though this is a weird case) ... might read better to say what it is like this or something: `A type to represent a request that doesn't return any JSON.`